application/entrypoint/helpers.go

```diff
@@ -12,10 +12,19 @@ import (
 
 func OnPanicRecover() {
 	if err := recover(); err != nil {
-		fmt.Println("🚨 Panicking 🚨")
+		panickingMsg := "🚨 Panicking 🚨"
+		fmt.Println(panickingMsg)
 		fmt.Println(err)
 		debug.PrintStack()
-		er := sentry.NewSentryErrorReporter(config.CurrentConfig(), nil)
+
+		c := config.CurrentConfig()
```

Comment:
my concern here is that if at some point config.CurrentConfig causes a panic.

Reply:
Hm. What would you recommend then?